modules/Bio/EnsEMBL/Compara/HAL/HALXS/Makefile-Cactus.PL

```diff
@@ -0,0 +1,51 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2021] EMBL-European Bioinformatics Institute
```

Comment:
Be aware in `release/104` we have changed the License header to avoid having to update them every year.